examples/40_advanced/example_plot_over_time.py

```diff
@@ -0,0 +1,75 @@
+"""
+==============================
+Plot the Performance over Time
+==============================
+
+Auto-Pytorch uses SMAC to fit individual machine learning algorithms
+and then ensembles them together using `Ensemble Selection
+<https://www.cs.cornell.edu/~caruana/ctp/ct.papers/caruana.icml04.icdm06long.pdf>`_.
+
+The following examples shows how to plot both the performance
+of the individual models and their respective ensemble.
+
+Additionally, as we are compatible with matplotlib,
+you can input any args or kwargs that are compatible with ax.plot.
+In the case when you would like to create multipanel visualization,
+please input plt.Axes obtained from matplotlib.pyplot.subplots.
+
+"""
+import warnings
+
+import numpy as np
+import pandas as pd
+
+from sklearn import model_selection
+
+from autoPyTorch.api.results_visualizer import PlotSettingParams
+from autoPyTorch.api.tabular_classification import TabularClassificationTask
+
+
+warnings.simplefilter(action='ignore', category=UserWarning)
+warnings.simplefilter(action='ignore', category=FutureWarning)
+
+
+if __name__ == '__main__':
```

Comment:
I think you can remove this `__main__` check as you are not using multiple jobs.